homeassistant/components/google_photos/services.py

```diff
@@ -53,6 +54,16 @@ def _read_file_contents(
                 translation_key="filename_does_not_exist",
                 translation_placeholders={"filename": filename},
             )
+        if filename_path.stat().st_size > CONTENT_SIZE_LIMIT:
+            raise HomeAssistantError(
+                translation_domain=DOMAIN,
+                translation_key="filename_too_large",
```

Comment:
```suggestion
                translation_key="file_too_large",
```
I think this is better for the translators, otherwise it could read as that the filename is too long